### DISCUSSION TITLE
Question about scenario parameters

### DISCUSSION THREAD

**Discussion by volm92** [InterDigitalInc/AdvantEDGE#294] [Q&A] [answered]
Is it possible to set the packet loss with a value less than one? I mean a decimal value. I tried to insert both the dot and the comma but I get an error.
The same for the bitrate. Is it possible to set a value less than one (decimal)? 
Or: is it possible to change the unit of measurement? In this way I could set bandwidth e.g. 500kbps 

Thannkyou!

**Comment by volm92** [comment]
I'm sorry, in packet loss I can do it, I cannot do in data rate.

**Comment by volm92** [comment]
Another question: how to get PacketLoss via API?
If I get scenario there is no packetloss...

**Reply by dilallkx** [reply]
Hi @volm92,

You can obtain the configured packet loss from the scenario, however there is no API to retrieve the measured values. We measure the RTT latency & UL/DL throughput and display them in the Network Metrics Dashboard of the AdvantEDGE frontend. Packet loss, however, is not included in this dashboard.

Please let us know if this is a feature you require for your application.
Regards!

**Comment by dilallkx** [comment, marked as answer]
Hi @volm92,

Packet loss supports decimal values.
Throughput supports integer values in the range of 1 - 1000000 Mbps (hardcoded).

I hope this clarifies the implementation.
Thanks!
